tools/cpplint.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
cpplint.py still only supports Python 2. This only works because we explicitly call it with python2 in the `tools/check_style.sh` file

Reply:
The upstream version of [`cpplint.py`](https://github.com/cpplint/cpplint/blob/develop/cpplint.py) supports Python 3.